src/spaces/utilities.jl

```diff
@@ -264,9 +264,10 @@ function randomwalk!(
     ifempty::Bool=true
 )
     θ = rand(polar)
-    direction = Tuple(rotate(SVector{agent.vel}, θ)) # assume vel has unit norm
+    r₀ = LinearAlgebra.norm(agent.vel)
+    direction = Tuple(rotate(SVector{agent.vel}, θ)) .* (r / r₀)
```

Comment:
This is problematic if `r₀=0`, which happens if at some point `randomwalk!` is called with step size `r=0`.
I should add a check to differentiate between the two cases `r₀=0` and `r₀!=0`; or `r=0` could be set to be invalid for `ContinuousSpace` but I assume there are cases where it can be convenient to have this option.

Reply:
nah, I would dissallow this option.

Reply:
Ok, now randomwalk in continuous space throws an error if r<=0